raft/storage_test.go

```diff
@@ -102,43 +94,27 @@ func TestStorageLastIndex(t *testing.T) {
 	s := &MemoryStorage{ents: ents}
 
 	last, err := s.LastIndex()
-	if err != nil {
-		t.Errorf("err = %v, want nil", err)
-	}
-	if last != 5 {
-		t.Errorf("last = %d, want %d", last, 5)
-	}
+	require.NoError(t, err)
+	require.Equal(t, uint64(5), last)
 
 	s.Append([]pb.Entry{{Index: 6, Term: 5}})
 	last, err = s.LastIndex()
-	if err != nil {
-		t.Errorf("err = %v, want nil", err)
-	}
-	if last != 6 {
-		t.Errorf("last = %d, want %d", last, 6)
-	}
+	require.Equal(t, nil, err)
```

Comment:
Sorry, I don't point it out in my last comment. I think it should be changed, and including the following lines.

Reply:
already changed!